test/ci-performance-test/travelSearch-expected-results-mc.csv

```diff
@@ -0,0 +1,61 @@
+tcId,transfers,duration,cost,walkDistance,startTime,endTime,details
```

Comment:
You can delete the 2 example files from this directory.

Reply:
Done